flocker/acceptance/test_api.py

```diff
@@ -84,6 +85,20 @@ def base_url(self):
             self.control_node.address, REST_API_PORT
         )
 
+    def _json_content(self, result, response_code):
```

Comment:
:question: Maybe think of a more meaningful method name. Maybe `check_and_decode_response`.